### ISSUE TITLE
VersionStore snapshot failure

### ISSUE THREAD

**Issue by ben10he** [manahl/arctic#414]
#### Arctic Version

```
1.43.0
```

#### Arctic Store

```
VersionStore
```

#### Platform and version

Using Python 3.5.2

#### Description of problem and/or code sample that reproduces the issue
I tried to snapshot a collection using function snapshot inside version_store. It produces following error message:

> <class 'pymongo.errors.OperationFailure'> list_symbols [Exceeded memory limit for $group, but didn't allow external sort. Pass allowDiskUse:true to opt in.], retrying 3

My understanding is that artic is trying to find all the symbols inside this collection, but since this is a large collections with over one million of symbols, artic function list_symbols causes above pymongo error. As the error message suggests, we should pass allowDiskUse:true instead. Is there any place in the arctic source code I can add this optional argument in? I tried to add a line 

`pipeline.extend({'allowDiskUse': True})`

inside function list_symbols in version_store.py, but it is not working.

Alternatively, is there any other way to solve this issue?

Many thanks.

**Comment by kraphtuos** [comment]
You need to pass the argument to aggregate.
Change line 165 to `results = self._versions.aggregate(pipeline, allowDiskUse=True)` should fix this.

If people are happy with this change I can add a PR for it.
@jamesblackburn @bmoscon @richardbounds

**Comment by bmoscon** [comment]
yeah I'm fine with it. Seems pretty straightforward. I assume pymongo/mongo will only use a disk if it HAS not, and not all the time.
